docs/advanced-features/persistence.md

```diff
@@ -0,0 +1,29 @@
+# Persistence for Static files
+
+Besides using an external shared storage (e.g. S3) for storing static files, it is also possible to leverage Kubernetes persistence. The following configuration creates a Persistent Volume Claim called `nautobot-static` and mounts it at the `/opt/nautobot/static` path of the Pods.
```

Comment:
```suggestion
Besides using an external shared storage (e.g. S3) for storing static files, it is also possible to leverage Kubernetes persistence. The following configuration creates a Persistent Volume Claim and mounts it at the `/opt/nautobot/static` path of the Pods.
```

It is not necessary that PVC name is `nautobot-static`

Reply:
Changed it